src/lib/helpers/string.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * Capitalizes the first letter of a string
+ *
+ * @export
+ * @param {string} str
+ * @returns {string}
+ */
+export function capitalize(str: string): string {
+    return str.charAt(0).toUpperCase() + str.slice(1);
+}
```

Comment:
It might be smarter to use CSS for this 🤔

Reply:
The behaviour is different. With CSS, the first letter of every word is capitalized. This function onnly does it for the first word. The question is, which one is preferred?

Reply:
iirc there should be a `:first-letter` selector 👀

Reply:
Oh, TIL! Will use CSS then, for sure :smile:

Reply:
@TorstenDittmann After further inspection, the way I see it, in order to use CSS I'd have to either pass in a custom prop to the inputs being used to Attribute, or use :global() in AttributeForm's styles. With this in mind, I think we're better off keeping this as is. What do you think?

Reply:
nah, wanted to make it simpler - this is not the way :-D